### ISSUE TITLE
Why heading is affecting paragraphs below?

### ISSUE THREAD

**Issue by timaschew** [jhildenbiddle/docsify-themeable#10]
Hi, first of all thanks for this beautiful theme.

I've realized that heading level 5 and 6 is affecting the styles of followed paragraphs.
![screen shot 2019-01-14 at 17 11 33](https://user-images.githubusercontent.com/110870/51124855-98e97400-181f-11e9-9bd8-367ca8213d0c.png)

By inspecting the styles I found this:
```
markdown-section h6, .markdown-section h6+p, .markdown-section h6+p+p {
    font-size: var(--heading-h6-font-size);
    color: var(--heading-h6-color, var(--heading-color));
}
```

That means next two paragraphs will have reduced font size, same as the heading (5 or 6) and the third paragraph is _normal_ then. I wonder by which conventions or design rules that was made. Why only 2 paragraphs? 

More important what if I want to show the second paragraph with a _normal_ font size, how do I reset the font-size for the 2nd paragraph without creating another heading (level 1-4)?

**Comment by jhildenbiddle** [comment]
Hi @timaschew.

The heading + paragraph styles you're referring to were intended to automatically match the small heading font size typically used for `<h5>` or `<h6>` elements to the content that followed. I was playing with this idea early on since markdown lacks syntax for creating `<small>` elements or applying any kind of style (like `font-size`) without adding HTML to the document.

In hindsight, this is not a great idea for several reasons. Perhaps most importantly, it just doesn't work. To be completely honest, I think I was experimenting with the idea and accidentally left this initial, very-basic approach in place.

Moving forward, I would propose this as the most obvious fix:

- Remove `font-size` from the `<p>` elements that follow `<h5>` and `<h6>` elements.
- Remove `font-color` from the `<p>` elements that follow `<h6>` elements. 

This more closely aligns with usual default heading styles provided by browsers. The downside is that if you want to make the text content that follows a small heading (like `<h5>` or `<h6`>), you're on your own to do something like this:

```markdown
##### Heading 5

<small>
This is small text
</small>

###### Heading 6

<div style="font-size: 0.65rem">
This is even smaller text.
</div>
```

Both of the above examples are valid markdown, but adding HTML to markdown kinda defeats the purpose of markdown (IMHO).

How do the above changes sound to you?

**Comment by timaschew** [comment]
Okay I see.

I agree it's what a pity that markdown doesn't support it, but I think using `<small>` is a good alternative.

> Moving forward, I would propose this as the most obvious fix:
> ...

Looks good to me :)

**Comment by jhildenbiddle** [comment]
This has been fixed in 0.6.1. A complete list of changes is available in the [changelog](https://github.com/jhildenbiddle/docsify-themeable/blob/master/CHANGELOG.md).

Thanks again for brining it to my attention, @timaschew.
